pkg/system/reconciler.go

```diff
@@ -273,20 +274,37 @@ func (r *Reconciler) Reconcile() (reconcile.Result, error) {
 	return res, nil
 }
 
+// bToMb convert bytes to megabytes
+func (r *Reconciler) bToMb(b uint64) uint64 {
+	return b / 1024 / 1024
+}
+
+// PrintMemUsage prints memory usage message.
+func (r *Reconciler) PrintMemUsage(phase string) {
+	var m runtimeStats.MemStats
+	runtimeStats.ReadMemStats(&m)
+	r.Logger.Infof("Memory Usage: Phase %s - Alloc = %v MiB  Sys = %v MiB  NumGC = %v", phase, r.bToMb(m.Alloc), r.bToMb(m.Sys), m.NumGC)
```

Comment:
Use `%q` in format to print quoted strings - very useful:
```suggestion
	r.Logger.Infof("Memory Usage: Phase %q - Alloc = %v MiB  Sys = %v MiB  NumGC = %v", phase, r.bToMb(m.Alloc), r.bToMb(m.Sys), m.NumGC)
```